packages/shared/styles/components/SfAddToCart.scss

```diff
@@ -0,0 +1,30 @@
+@import '../variables';
+
+.sf-add-to-cart {
+  display: flex;
+
+  &__button {
+    &:disabled {
+      background-color: $c-accent-secondary;
+      cursor: default;
+    }
+  }
+
+  &__select-quantity {
+    margin-left: 0.5rem;
+
+    input {
+      padding-left: 0.5rem;
+      padding-right: 0.5rem;
+      width: auto;
+      height: 100%;
+      max-width: 90px;
```

Comment:
meybe we need scss variable for it?

Reply:
for `max-width`? Sure :)

Reply:
for my yes but if you think is not necessary then feel free :)